client/inbox-panel/dissmiss-all-modal.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { Button, Modal } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const DissmissAllModal = ( { setShowDismissAllModal, dismissAllNotes } ) => {
+	const title = __( 'Dismiss all messages', 'woocommerce-admin' );
```

Comment:
Do these need to be variables or could we put these inline below?  It becomes a bit more time consuming to read scanning up and down.

Reply:
Good suggestion! Updated in https://github.com/woocommerce/woocommerce-admin/pull/7871/commits/14ca0f6f14d0578d70523051d4511bbabb2283ef